yaml/github-actions/security/third-party-action-not-pinned-to-commit-sha.test.yml

```diff
@@ -54,6 +54,13 @@ jobs:
       # ruleid: third-party-action-not-pinned-to-commit-sha
       - uses: docker://alpine:3.8
 
+      # ok: third-party-action-not-pinned-to-commit-sha
+      - name: Upload SARIF file for GitHub Advanced Security Dashboard
+        uses: github/codeql-action/upload-sarif@e4262713b504983e61c7728f5452be240d9385a7
```

Comment:
Should this test line be @v2 instead? Otherwise I think the hash would keep it from flagging even without the change. (But please ignore this if I'm wrong!)

Reply:
No, you are totally right, that was an oversight on my part. I updated it!